### PR TITLE
testing removing ntp server

### PR DESCRIPTION
**Pull request by kbrock**
https://github.com/ManageIQ/manageiq/pull/20623

### CODE REVIEW

**Comment by kbrock** [comment]
this keeps failing for me in sporadic and what seem unrelated ways.
frustrating

**Comment by d-m-u** [comment]
I don't think `- TEST_REPO=manageiq-automate-models` is what you're looking for here.

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,15 @@ matrix:
...
-  - REPOS=
+  - REPOS=ManageIQ/manageiq#20623
   matrix:
-  - TEST_REPO=manageiq
+  - TEST_REPO=manageiq-api
+  - TEST_REPO=manageiq-appliance  
+  - TEST_REPO=manageiq-ui-classic
+  - TEST_REPO=manageiq-content
+  - TEST_REPO=manageiq-automation_engine
+  - TEST_REPO=manageiq-automate-models
```
Comment (bdunne) [inline .travis.yml:24]: Did you mean `manageiq-content`?

Comment (kbrock) [reply .travis.yml:23]: dropped manageiq-content. it had ntp in a comment somewhere

**Comment by kbrock** [comment]
thanks @d-m-u for getting me all set.

I was surprised not to see "these are not the droids you are looking for" reference - but then I realized the two people on this thread

@bdunne does this now look good to you?

**Comment by d-m-u** [comment]
I think you can close this, Brandon merged 20623 this morning.

**Pull request closed without merging**